### ISSUE TITLE
[BUG] Keyboard tabs can't access accordions or expand/collapse anchor elements

### ISSUE THREAD

**Issue by Atticus29** [Symbiota/Symbiota#3362]
# Required

## Summary

In order to be compliant with https://www.w3.org/WAI/WCAG22/quickref/?versions=2.1&showtechniques=231%2C321, we need the keyboard to be able to navigate across all actionable elements. Some are missing on the search/index.php page, and this is a highly trafficked page.

<img width="1582" height="951" alt="Image" src="https://github.com/user-attachments/assets/7b65c108-13ab-4cf5-a475-488a1eec59ee" />

## Github Issue/Projects Fields

- [x] Populate label(s) (e.g., bug, etc.)
- [x] Populate the GitHub Projects fields (e.g., status, priority, affected branch, etc.)
- [ ] If relevant, add reference to original osTicket: 

# Optional

## Environment Details

**Symbiota Portal URL:**

**Browser:**

**Operating System:** 

**Device:** 

## Steps to Reproduce

2. Click on '...'
3. Enter '...'

## Error Messages

```
Paste error message here
```

**Comment by Atticus29** [comment]
@Atticus29 needs to update https://docs.google.com/document/d/1bjdn-46e_OBOATvEr8DoZiu1kBMWd_bxpzcSv5lcVBk/edit?tab=t.0 after this is completed.
